community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,440 @@
+    The Aqua Security Operator runs within a Kubernetes or Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components as
+    * Server (Console)
+    * Gateway
+    * Database (Not recommended for production environments)
+    * Enforcer (Agent)
+    * Scanner CLI
+    * CSP (Package of Server, Gateway and Database)
+
+    Use the aqua-operator to - 
+    * Deploy Aqua Security components on Kubernetes or Openshift
+    * Scale up Aqua Security components with extra replicas
+    * Assign metadata tags to Aqua Security components
+    * Automatic scale to the Aqua Scanner CLI by the count of the Scanning Queue.
+
+    ## Before You Begin
+    ### Kubernetes 1.11+
+    the operator in kubernetes able to create the all the requirments by passing the parmater **requirments** as `true`
+    but you can also create the requirments - 
+      * Service account with pull permissions from aqua docker registry
+      * Secret for docker registry 
+      * Secret for database
+
+    ## After You Install
+    Once the operator is installed in the cluster, you now can use the crds to install aqua cluster and components.
+  keywords: ['aqua-security', 'scanning', 'security', 'runtime-security']
+  maintainers:
+  - email: nissim.bitan@aquasec.com
+    name: Nissim Bitan, Aqua Security
+  provider: 
+    name: Aqua Security, Inc.
+  maturity: alpha
+  labels:
+    name: aqua-operator
+  selector:
+    matchLabels:
+      name: aqua-operator
+  links:
+  - name: Aqua Security
+    url: https://www.aquasec.com/
+  - name: Prometheus Operator
+    url: https://github.com/aquasecurity/aqua-operator
+  icon:
+  - base64data: PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDIzLjAuMiwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCAxMDAgMTAwIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCAxMDAgMTAwOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+CjxzdHlsZSB0eXBlPSJ0ZXh0L2NzcyI+Cgkuc3Qwe2ZpbGw6I0YwREUzNDt9Cgkuc3Qxe2ZpbGw6I0Y5QUU0MTt9Cgkuc3Qye2ZpbGw6Izk5RDVFQjt9Cgkuc3Qze2ZpbGw6IzAyQjBENDt9Cgkuc3Q0e2ZpbGw6I0Y1OTMyMDt9Cgkuc3Q1e2ZpbGw6IzA5ODA5QTt9Cjwvc3R5bGU+CjxnPgoJPHBvbHlnb24gY2xhc3M9InN0MCIgcG9pbnRzPSI1MCwyLjEgMjUuNSwyNi4xIDc0LjUsMjYuMSA3NC41LDIuMSAJIi8+Cgk8cG9seWdvbiBjbGFzcz0ic3QxIiBwb2ludHM9Ijc0LjUsNzMuOSA5OSw1MCA5OSwyNi4xIDc0LjUsMjYuMSAJIi8+Cgk8cG9seWdvbiBjbGFzcz0ic3QyIiBwb2ludHM9IjI1LjUsNzMuOSAyNS41LDk3LjkgNTAsOTcuOSA3NC41LDczLjkgCSIvPgoJPHBvbHlnb24gY2xhc3M9InN0MyIgcG9pbnRzPSIxLDUwIDEsNzMuOSAyNS41LDczLjkgMjUuNSwyNi4xIAkiLz4KCTxyZWN0IHg9Ijc0LjUiIHk9IjIuMSIgY2xhc3M9InN0NCIgd2lkdGg9IjI0LjUiIGhlaWdodD0iMjMuOSIvPgoJPHJlY3QgeD0iMSIgeT0iNzMuOSIgY2xhc3M9InN0NSIgd2lkdGg9IjI0LjUiIGhlaWdodD0iMjMuOSIvPgo8L2c+Cjwvc3ZnPgo=
+    mediatype: image/svg+xml
+  installModes:
+  - supported: false
```

Comment:
The `OwnNamespace` installMode should always be set to true. This causes the operator to fail CI: https://travis-ci.com/operator-framework/community-operators/builds/106452514#L2803
@ecordell